### PR TITLE
Add feature to check out different address books

### LINKED ISSUE #122: As a user, I can have multiple applicant lists and checkout between them
... so that I can group my applicants neatly

### PR DESCRIPTION
**Pull request by deeyonn**
Resolves #122 

* Modified method signature of Command::execute to include Model
* Added StorageStub to AddCommandTest.java and CommandTestUtil.java
* Added case for CheckoutCommand in AddressBookParser.java
* Added parseAddressBookFile to ParserUtil.java
* Added setAddressBookStorage to Storage.java and StorageManager.java
* Added AddressBookFile.java
* Added CheckoutCommand.java
* Added CheckoutCommandParser.java

### CODE REVIEW

**Review by cliftonfelix — COMMENTED** [review]
Is there a way to reload the app once the checkout command is called? or the least we can do is maybe tell the user that the app will be closing in a few seconds and they need to reopen, because rn after I checkout, if I don't reload directly, I can still edit the current addressbook

File: src/main/java/seedu/address/logic/commands/CheckoutCommand.java [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        requireNonNull(storage);
+
+        if (model.getAddressBookFilePath().equals(addressBookFilePath)) {
+            throw new CommandException(MESSAGE_SAME_BOOK);
+        }
+
+        logger.info("----------------[addressBookFilePath][" + addressBookFilePath + "]");
+
+        UserPrefsStorage userPrefsStorage = new JsonUserPrefsStorage(storage.getUserPrefsFilePath());
+        UserPrefs userPrefs = initPrefs(userPrefsStorage);
```
Comment (cliftonfelix) [inline src/main/java/seedu/address/logic/commands/CheckoutCommand.java:67]: Add `userPrefs.setAddressBookFilePath(addressBookFilePath);` below line 67

**Review by cliftonfelix — COMMENTED** [review]
Add `statusbarPlaceholder.getChildren().add((new StatusBarFooter(logic.getAddressBookFilePath())).getRoot());` in `executeCommand` method in `MainWindow.java`

**Review by cliftonfelix — APPROVED** [review]
LGTM

**Review by JasonCP14 — APPROVED** [review]

**Pull request merged**